consensus/src/dag/dag_handler.rs

```diff
@@ -67,10 +66,39 @@ impl NetworkHandler {
         _buffer: Vec<DAGMessage>,
     ) -> SyncOutcome {
         // TODO: process buffer
+
+        // TODO: Pass bounded executor from EpochManager.
+        let executor = BoundedExecutor::new(2, Handle::current());
+        let epoch_state = self.epoch_state.clone();
+
+        let mut verified_msg_stream = concurrent_map(
+            dag_rpc_rx,
+            executor,
+            move |rpc_request: IncomingDAGRequest| {
+                let epoch_state = epoch_state.clone();
+                async move {
+                    debug!("deserializing and verifying message");
+                    let result = rpc_request
+                        .req
+                        .try_into()
+                        .and_then(|dag_message: DAGMessage| {
+                            monitor!(
+                                "dag_message_verify",
+                                dag_message.verify(rpc_request.sender, &epoch_state.verifier)
+                            )?;
```

Comment:
Who handles this error?

Reply:
process_verified_message in dag_handler.